html/css/graph.min.css

```diff
@@ -0,0 +1 @@
+#banner{width:98%;text-align:center;font-weight:700;color:#fff;border-radius:5px;padding-top:.3em;padding-bottom:.2em}#cell_product_image{text-align:center;width:23%}#prod_ref_image{max-width:99%}#links_off{text-align:center;vertical-align:bottom;padding:0;margin:0}#links_off>div{padding-top:.2em}#links_off a,a{text-decoration:none;color:#09f;font-weight:700}#link_json_off{visibility:hidden}@media screen and (min-width:12in){#link_json_off{visibility:visible}}#product_details{width:auto;height:2.2em;font-size:larger;text-align:center;overflow-y:hidden}#prod_ref_code{font-weight:700}#criteria{font-size:inherit;text-align:right;padding:0;margin:0;min-width:99%}#criteria *{vertical-align:middle}.barcode_product_selected{background-color:#09f!important}.barcode_no_selection{background-color:none}#msg_warning_prod_ref{font-size:x-large;color:red;margin-right:2ex}td>select{font-size:inherit;width:18em}#input_product_code{font-size:inherit;width:8em}#panel_input_code{font-size:inherit;display:flex;float:left}#graph{text-align:center}#svg_graph{font-size:medium}.axis line,.axis path{fill:none;stroke:grey;stroke-width:1;shape-rendering:crispEdges}svg>g:first-child>g:first-child text{display:none}div.tooltip{position:absolute;text-align:center;width:450px;height:140px;padding:2px;font:10px sans-serif;background:#b0c4de;border:0;border-radius:8px;pointer-events:none}#server_log_cell>*{display:none!important}#products_suggestion{margin:5px;background:#a3cbe0;border:2px solid #000;box-shadow:0 0 10px #000}#products_suggestion ul{margin:0;padding:0;height:250px;display:table;white-space:nowrap;list-style:none}#products_suggestion li{display:table-cell;text-align:center;padding-left:5px;padding-right:5px}.cell_suggestion{float:left;padding-left:2px;padding-right:2px;width:14%;height:auto;border:solid thin #09f}img.grade_border{border:18px solid #e63e11}.product_selected{border:18px solid #09f}#menu_selection{visibility:visible;background-color:#09f}#menu_selection>div{width:15%;float:left;text-align:center;vertical-align:middle;font-size:large;color:#fff}#menu_selection img{height:85px;padding-top:2%;padding-bottom:1%;padding-left:5%;padding-right:5%}#nb_suggestions{padding-top:2%;font-size:xx-large}.detailsProduct{font-size:larger;text-align:center;vertical-align:middle;position:absolute;padding-top:2%;background-color:silver;width:0;height:0;z-index:10}.table_sel_prod{width:99%}.sel_prod_img{margin-top:1%;margin-bottom:1%;text-align:right}.sel_prod_header{text-align:center;padding-left:1%}.sel_prod_name{color:#1e90ff}.sel_prod_similarity{font-weight:700;color:purple}.close_details_sel_prod{position:absolute;left:0;width:100%;background-color:#2cb4da;color:#fff;font-weight:700;font-size:xx-large;margin-top:1%;padding-top:2%;padding-bottom:1%}
```

Comment:
This file does not need to be added to git, because the minified version `html/css/dist/graph.js` is automatically generated by gulp.

Reply:
indeed.. has been removed in commit/push I have just made now.